noggin/controller/user.py

```diff
@@ -139,12 +140,6 @@ def user_settings_otp(ipa, username):
             'otptoken_find', [], {'ipatokenowner': username, 'all': True}
         )['result']
     ]
-    enabled_count = 0
```

Comment:
Maybe I misunderstood the conversation but won't freeipa reject the disabling of the last token when the auth policy is 2fa? We should probably revert to the flash message instead to allow the cases where they can disable the last token.

Reply:
Ok, have tested two possibilities here. 

1. on the freeIPA setup, i checked the Default Authentication types to be both "Password" and "Two factor authentication (password + OTP)". this allowed the following (note that this is just for logging into freeIPA webui / noggin, not kinit, which i didnt test)
 * without an otp token setup, i log in with just password.
 * with an otp token setup, i cannot log in with just password, i have to use password + OTP.
 * i can disable / delete the last OTP token, and then just log in with password again

2. on the freeIPA setup, i checked the Default Authentication types to be just"Two factor authentication (password + OTP)". this allowed the following (note that this is just for logging into freeIPA webui / noggin, not kinit, which i didnt test)
* before adding an otp token, i can log in with just password.
* when i add an otp token, i then have to log in with password  + OTP
* i *cannot* disable or delete the last OTP token. so i can never go back to just using a password.

Reply:
Assuming that we are setting up as option 2 above, as-is, if you try to disable or delete the last token, you get a flash message saying that it can't be done. Will work on some better flash messaging here, catching the error where we can't delete / disable the last one.